### DISCUSSION TITLE
"instanceof_function_ex" - Problem when Installing Pimcore Skeleton on Server

### DISCUSSION THREAD

**Discussion by jvolkmib** [pimcore/pimcore#9467] [Q&A / Support]
When i trying to install Pimcore on my Webserver over ./vendor/bin/pimcore-install, i get the Error-Message: 
`ld-elf.so.1: /usr/iports/php80/lib/php/20200930/imagick.so: Undefined symbol "instanceof_function_ex"

i think Pimcore Call the Function instanceof_function_ex, but in PHP 8.X the Function changed to instanceof_function.

Has anybody a same Problem ? 

Thanks a Lot 

Johannes 
`

**Comment by NiklasBr** [comment]
You need a newer version of imagick, it can be installed [using these instructions](https://github.com/Imagick/imagick/issues/358#issuecomment-737385312)
